Vagrantfile

```diff
@@ -44,6 +44,25 @@ Vagrant.configure("2") do |config|
     if host_arch == "x86_64"
       lv.machine_type = "q35"
     end
+    # The VM needs one additional virtio-net device and iommu enabled
+    # for the vfio tests.
+    if job == "VFIO" and host_arch == "x86_64"
+      # Overwrite the default -machine already built by vagrant as
+      # additional arguments aren't allowed on `machine_type` property.
```

Comment:
I'm not sure what you mean by this.  It's definitely possible to have multiple `-machine` arguments (and frequently useful to give machine arguments separately from the machine type).

Reply:
I need to add `kernel-irqchip=split` to the machine type. Setting setting `lv.machine_type=q35,kernel-irqchip=split` won't work as it seems the vagrant-libvirt plug-in cannot parse correctly the comma-separated string.

I wasn't sure I could give `-machine` multiple times and QEMU would sum up the arguments (in my mind the last `-machine`'s value would overwrite all the previous)

So a command-line like `-machine pc-q35-5.1,accel=kvm,usb=off,dump-guest-core=off (...) -machine=q35,kernel-irqchip=split` is equivalent to `-machine pc-q35-5.1,accel=kvm,usb=off,dump-guest-core=off,kernel-irqchip=split`?

Reply:
Well, yes, but more to the point, so is `-machine pc-q35-5.1 -machine accel=kvm -machine usb=off -machine dump-guest-core=off -machine kernel-irqchip=split`.